feature/home/src/main/java/com/mashup/dorabangs/feature/home/HomeRoute.kt

```diff
@@ -88,6 +152,7 @@ fun HomeRoute(
             navigateToClassification = navigateToClassification,
             navigateSaveScreenWithoutLink = navigateToSaveScreenWithoutLink,
             navigateToHomeTutorial = navigateToHomeTutorial,
+            requestUpdate = { viewModel.updatePost(it) },
```

Comment:
```suggestion
            requestUpdate = viewModel::updatePost,
```
주소값을 전달해주면 불필요한 리컴포지션을 방지한다고 하는데, 봤던 블로그가 있는데 어딨는지 모르겠네;